tests/test_utils.py

```
@@ -2,325 +2,304 @@
-    
-    @patch('subprocess.run')
+
+    @patch("subprocess.run")
     def test_run_command_exception_handling(self, mock_run):
         """Test command execution exception handling"""
         mock_run.side_effect = Exception("Unexpected error")
-        
-        rc, stdout, stderr = run_command('bad-command')
-        
+
+        rc, stdout, stderr = run_command("bad-command")
```

Ruff (0.13.1)

Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)